runtime/Cargo.toml

```diff
@@ -1,26 +1,26 @@
 [package]
 name = "sapphire-paratime"
-version = "0.1.0"
+version = "0.1.0-testnet"
 authors = ["Oasis Protocol Foundation <info@oasisprotocol.org>"]
 edition = "2021"
 license = "Apache-2.0"
-resolver = "2"
 
 [package.metadata.orc.release]
-runtime-id = "000000000000000000000000000000000000000000000000e2eaa99fc008f87f"
+runtime-id = "000000000000000000000000000000000000000000000000a6d1e3ebf60dff6c"
 
 [package.metadata.orc.test]
-runtime-id = "00000000000000000000000000000000000000000000000072c8215e60d5bca7"
+runtime-id = "0000000000000000000000000000000000000000000000000000000000000000"
```

Comment:
This is the ID used on testnet, so it should be the one above. The mainnet one is not yet defined, so just switch the two.